java/arcs/sdk/wasm/WasmJsInterop.kt

```diff
@@ -249,3 +249,11 @@ fun log(msg: String) {
     write(msg.toWasmString())
     flush()
 }
+
+fun combine(vararg handles: WasmSingletonImpl<*>): WasmCombinedSingletonHandle<*> {
+    val l = mutableListOf<WasmSingletonImpl<WasmEntity>>()
+    for (handle in handles) {
+        l.add(handle as WasmSingletonImpl<WasmEntity>)
+    }
+    return WasmCombinedSingletonHandle(l)
```

Comment:
This can just be:

```
fun combine(vararg handles: WasmSingletonImpl<*>) = WasmCombinedSingletonHandle(
    handles.toList()
)
```

Reply:
I think this naming is confusing in a sense because similar operators in Rx and Flow with the 'combine' work by giving you an N-ary callback.  I think the problem with this API is that when you get an entity, you don't know what handle it is from, and so you'd need a when(entity) {} matcher to check the type. 

if instead, you have a method like this
```

fun <E1 : Entity, E2: Entity, 
        H1: Handle<E1>, H2: Handle<E2>>
combine(handle1: H1, handle2: H2, block: (E1,E2) -> Unit) = handle1.asFlow().combine(handle2.asFlow(), block)

fun <E:Entity, H: Handle<E>> H.asFlow() = callbackFlow {
   this.onUpdate { entity -> offer(entity) }
}
```

You could then write

```
combine(handle1, handle2) { entity1, entity2 -> doSomething(entity1, entity2) }
```